lib/bolt/transport/winrm.rb

```diff
@@ -8,14 +8,19 @@ module Bolt
   module Transport
     class WinRM < Base
       def self.options
-        %w[port user password connect-timeout ssl ssl-verify tmpdir cacert extensions interpreters]
+        %w[
+          port user password connect-timeout ssl ssl-verify tmpdir cacert
+          extensions interpreters file-protocol smb-port
+        ]
       end
 
       def self.default_options
         {
           'connect-timeout' => 10,
           'ssl' => true,
-          'ssl-verify' => true
+          'ssl-verify' => true,
+          'file-protocol' => 'winrm',
+          'smb-port' => 445
```

Comment:
I think we should keep this default with the winrm port defaults. https://github.com/puppetlabs/bolt/blob/71aff3e06e3a6232b14f76d99af91edbf6d53570/lib/bolt/transport/winrm/connection.rb#L29-L30

Reply:
Oh, you do set that in `::WinRM::Connection`. You can just delete the `smb-port` default here I think.